easybuild/easyconfigs/g/Gctf/Gctf-1.06.eb

```diff
@@ -0,0 +1,25 @@
+name = 'Gctf'
+version = '1.06'
+
+homepage = 'https://www.mrc-lmb.cam.ac.uk/kzhang/'
+description = """Gctf: real-time CTF determination and correction, Kai Zhang, 2016"""
+
+toolchain = SYSTEM
+
+source_urls = ['http://www.mrc-lmb.cam.ac.uk/kzhang/Gctf/']
+sources = [
+    '%(name)s_v%(version)s_and_examples.tar.gz',
+]
+patches = [('Gctf-wrapper.sh', '.')]
```

Comment:
@akesandgren This line should be removed now the custom easyblock is used?

Reply:
Gah, I thought I had done that... fixed.